common/src/main/java/com/microsoft/identity/common/internal/telemetry/TelemetryEventStrings.java

```diff
@@ -192,6 +192,8 @@
         public static final String LOCAL_ACQUIRE_TOKEN_SILENT= "103";
```

Comment:
Is `LOCAL_COMPLETE_ACQUIRE_TOKEN_INTERACTIVE` supposed to be "102"?

Reply:
Let's hold for @shahzaibj -- can't locate this information